sql/handler.h

```diff
@@ -331,7 +331,10 @@ enum enum_alter_inplace_result {
 
 /* Support native hash index */
 #define HA_CAN_HASH_KEYS        (1ULL << 58)
-#define HA_LAST_TABLE_FLAG HA_CAN_HASH_KEYS
+
+/* Has persistent count enabled */
+#define HA_PERSISTENT_COUNT  (1ULL << 59)
```

Comment:
I think that we should try to reuse the existing flag `HA_HAS_RECORDS` inside InnoDB, instead of introducing a similar flag.
The challenge is that at the time `ha_innobase::table_flags()` or `ha_innobase::m_int_table_flags` are called, the data file might not have been opened in InnoDB yet. Could we make the `table_flags()` change after `open()`?